app/build.gradle

```diff
@@ -24,30 +24,34 @@ android {
     }
 
     buildTypes {
+        buildTypes {
```

Comment:
Oh, sorry, my bad, I am amazed that this is even working :) I'll fix it in my free time